jsk_robot_common/jsk_robot_startup/config/install_systemd.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+jsk_robot_startup=$(builtin cd "`dirname "${BASH_SOURCE[0]}"`"/.. > /dev/null && pwd)
+if [ $# != 1 ]; then
+    echo "Invalid number of arguments"
+    exit 1
+fi
+
+# Copy scripts
+cd $jsk_robot_startup/systemd
+if [ ! -d /usr/local/bin/jsk_robot_startup ]; then
+  sudo mkdir -p /usr/local/bin/jsk_robot_startup
+fi
+for file in $(ls *.sh); do
+    sudo cp $file /usr/local/bin/jsk_robot_startup
+    sudo chown root:root /usr/local/bin/jsk_robot_startup/$file
+    sudo chmod 755 /usr/local/bin/jsk_robot_startup/$file
+    echo "copied jsk_robot_startup/systemd/$file to /usr/local/bin/jsk_robot_startup/$file"
+done
+echo "device: $1"
+echo 3idevice=$1|sudo xargs -i sed -i {} /usr/local/bin/jsk_robot_startup/playsound.sh
+sudo ln -sf /usr/local/bin/jsk_robot_startup/playsound.sh /usr/local/bin/jsk_robot_startup/start_sound.sh
+sudo ln -sf /usr/local/bin/jsk_robot_startup/playsound.sh /usr/local/bin/jsk_robot_startup/stop_sound.sh
+
+# Enable systemd service
+for file in $(ls *.service); do
+    sudo cp $file /etc/systemd/system
+    sudo chown root:root /etc/systemd/system/$file
+    sudo chmod 644 /etc/systemd/system/$file
+    echo "copied jsk_robot_startup/systemd/$file to /etc/systemd/system/$file"
+    sudo systemctl enable $file
+done
+
+# Default sound
+# Created by VOICEVOX:四国めたん
+cd $jsk_robot_startup/data
+if [ ! -d /usr/share/sounds/jsk_robot_startup ]; then
+  sudo mkdir -p /usr/share/sounds/jsk_robot_startup
```

Comment:
Can you change the directory to `/usr/local/share/sounds/jsk_robot_startup`?

https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html says

> Locally installed software must be placed within /usr/local rather than /usr unless it is being installed to replace or upgrade software in /usr. [[28]](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html#ftn.idm236091898144)